storybook/src/server/framework-preset.ts

```diff
@@ -65,7 +80,10 @@ export const webpack: StorybookConfig['webpack'] = async (config, options) => {
                           projectDir: symfonyOptions.projectDir,
                           additionalWatchPaths: symfonyOptions.additionalWatchPaths,
                       }),
-                TwigLoaderPlugin.webpack({ twigComponentConfiguration: symfonyOptions.twigComponent }),
+                TwigLoaderPlugin.webpack({
+                    twigComponentConfiguration: symfonyOptions.twigComponent,
+                    twigConfiguration: symfonyOptions.twig,
```

Comment:
It looks like you don't need to pass the twig configuration here, as you built all the paths in `getBuildOptions`

Reply:
indeed everything works without it. Thanks and done